[tflite] Add XNNPACK delegate for v2.3.0

anyj0527 opened the pull request:
- Add XNNPACK delegate for v2.3.0
- Static libs of XNNPACK, pthreadpool, clog, cpuinfo are also included

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

myungjoo commented:
@jaeyun-jung : It's your call.

jaeyun-jung commented on external/jni/Android-tensorflow-lite-2.3.0.mk:
```diff
@@ -32,11 +41,13 @@ CORE_CC_ALL_SRCS := \
     $(wildcard $(TF_LITE_DIR)/kernels/internal/*.cc) \
     $(wildcard $(TF_LITE_DIR)/kernels/internal/optimized/*.cc) \
     $(wildcard $(TF_LITE_DIR)/kernels/internal/reference/*.cc) \
+    $(wildcard $(TF_LITE_DIR)/profiling/*.cc) \
```
Q: is this line mandatory for tf-lite or xnnpack?

anyj0527 replied:
@jaeyun-jung PTAL

anyj0527 committed:
[tflite] Add XNNPACK delegate for v2.3.0

- Add XNNPACK delegate for v2.3.0
- Static libs of XNNPACK, pthreadpool, clog, cpuinfo are included

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

jaeyun-jung commented on external/jni/Android-tensorflow-lite-2.3.0.mk:
```diff
@@ -89,6 +89,10 @@ CORE_CC_ALL_SRCS += \
 
 TF_LITE_FLAGS += -DEGL_EGLEXT_PROTOTYPES -DCL_TARGET_OPENCL_VERSION=220
 
+# XNNPACK delegate
```
recommend to move this to line 54 for the readability :)

anyj0527 replied:
cool!
